backends/xnnpack/test/tester/tester.py

```diff
@@ -496,14 +496,14 @@ def generate_random_inputs(self):
                         # the function to compute the derived dim.
                         dim_spec = dim_spec.root
                         fn = dim_spec.fn
-                    elif isinstance(dim_spec, torch.export.dynamic_shapes._Dim):
+                    elif isinstance(dim_spec, torch.export.dynamic_shapes.Dim):
```

Comment:
did this change in pytorch?

Reply:
looks like in https://github.com/pytorch/pytorch/pull/149891/